x-pack/solutions/observability/plugins/synthetics/public/apps/synthetics/components/common/components/error_callout.tsx

```diff
@@ -0,0 +1,56 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+import { EuiCallOut } from '@elastic/eui';
+import { i18n } from '@kbn/i18n';
+import React from 'react';
+import { IHttpSerializedFetchError } from '../../../state';
+
+/**
+ * Use this component when displaying fetch-related errors.
+ */
+export function ErrorCallout(error: IHttpSerializedFetchError<unknown>) {
+  return (
+    <EuiCallOut
```

Comment:
@maciejforcone @nastasha-solomon 

What are your thoughts about showing the error here like this:

![image](https://github.com/user-attachments/assets/b75d621d-1f93-4fb9-a3d2-fc8470d003b7)